src/libs/actions/ReportActions.js

```diff
@@ -0,0 +1,73 @@
+import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as CollectionUtils from '../CollectionUtils';
+
+/**
+ * Map of the most recent non-loading sequenceNumber for a reportActions_* key in Onyx by reportID.
+ *
+ * What's the difference between reportMaxSequenceNumbers and reportActionsMaxSequenceNumbers?
+ *
+ * Knowing the maxSequenceNumber for a report does not necessarily mean we have stored the report actions for that
+ * report. To optimize and understand which reportActions we need to fetch we also keep track of the max sequenceNumber
```

Comment:
NAB; I think switching "understand" and "optimize" reads a bit clearer.

```suggestion
 * report. To understand and optimize which reportActions we need to fetch we also keep track of the max sequenceNumber
```